package.json

```diff
@@ -25,23 +25,19 @@
     "yargs": "^15.3.1"
   },
   "devDependencies": {
-    "@elastic/eslint-config-kibana": "^0.15.0",
     "@mapbox/geojson-rewind": "^0.5.1",
     "ajv": "^8.3.0",
     "ajv-formats": "^2.1.0",
     "babel-eslint": "^10.1.0",
     "csv-parse": "^4.12.0",
-    "eslint": "^6.8.0",
+    "eslint": "^7.32.0",
     "eslint-plugin-babel": "^5.3.0",
     "eslint-plugin-import": "^2.20.2",
-    "eslint-plugin-jest": "^23.8.2",
-    "eslint-plugin-mocha": "^6.3.0",
     "eslint-plugin-prefer-object-spread": "^1.2.1",
-    "eslint-plugin-react": "^7.19.0",
```

Comment:
💯  I think I originally just copied all the `eslint-*` dependencies from kibana even though we weren't using them. 😅

Reply:
haha, no prob! :+1: